.editorconfig

```
@@ -0,0 +1,31 @@
+root = true
+
+[*]
+charset = utf-8
+end_of_line = lf
+trim_trailing_whitespace = true
+max_line_length = 120
+ij_visual_guides = 100,120,150
+
+[{go.mod,go.sum,*.go}]
+insert_final_newline = true
```

I don't get why you removed the editorconfig checker via 9bc8f424c5afb25f5f87a129375e5672e181afb5

I would have expected you to keep it and add here what was needed

This PR is about adding editorconfig and now all call to editorconfig will fail

```suggestion
max_line_length = off # disabled because golangci-lint do it in a better way
insert_final_newline = true
```

---

revert

---

Thanks